### DISCUSSION TITLE
Where can I find the source code of the FlaskApp interface?

### DISCUSSION THREAD

**Discussion by rukasudev** [vanna-ai/vanna#459] [Q&A]
Hi everyone! Great project! ✨

I have a question: Where can I find the source code of this file? I want to make some custom changes.
https://github.com/vanna-ai/vanna/blob/main/src/vanna/flask/assets.py

**Comment by Molrn** [comment]
I haven't been able to find it anywhere, my guess is that it could be generated from a no code application. @zainhoda It would be a huge help for contributors and newcomers to have an access to these assets instead of a raw export. If you want to keep it this way, could you please indicate how these exports are generated so that people could contribute or modify it on their side ?
